.pylintrc

```diff
@@ -23,7 +29,23 @@ disable=
  old-style-class,
 # "R" Refactor recommendations
  too-few-public-methods,
+#  disable warning when local pylint disable is used
+ locally-disabled,
 
 [REPORTS]
-
 msg-template='[{msg_id} {symbol}] {msg} File: {path}, line {line}, in {obj}'
+
+[FORMAT]
+
+# Maximum number of characters on a single line.
+max-line-length=119
+
+[LOGGING]
+
+# Format style used to check logging format string. `old` means using %
+# formatting, `new` is for `{}` formatting,and `fstr` is for f-strings.
+logging-format-style=old
```

Comment:
These are related to the, eg: ` loggerinst.info("Received return code: %s\n" % str(ret_code))` 

py26 allow % but py27 allow only ` loggerinst.info("Received return code: %s\n", str(ret_code))`

So, these config make py26_lint and py27_lint happy.

Reply:
Then I don't understand, why are you doing all the changes in `convert2rhel/pkghandler.py`.

There, you're changing `loggerinst.info("Received return code: %s\n" % str(ret_code))` to `loggerinst.info("Received return code: %s\n", str(ret_code))`.

If you change all the occurrences of `%` to `,`, why do we need new style then?

Reply:
I believe I give you wrong answer, sorry... the reason of old style is: on py27 force us to use `"abcd {0}".format()`  but py26 doesn't have the `format` on str... so.. we need these old style to pylint27 not warning if we not use `format`.

From the doc:

> logging-format-style is a new option for the logging checker for usage of str.format() style format strings in calls to loggers.
> 
> It accepts two options: --logging-format-style=old for using % style formatting, which is the assumed default, and --logging-format-style=new for using {} style formatting.
>

Reply:
and the Changes from `loggerinst.info("Received return code: %s\n" % str(ret_code))` to `loggerinst.info("Received return code: %s\n", str(ret_code))` is related to another warning massage, these one:

> [W1201 logging-not-lazy] Specify string format arguments as logging function parameters 

not for old style warning.

I'll focus more in fix the lint errors instead of warnings, It's god to have all clean, but I believe we can handle with some warnings for now, right?

Reply:
Yes, we can deal with warnings later. Focus on errors first.

Anyway, it seems to me that if we change all the logging calls to use comma instead of `%`, we kill two birds with one stone - 1) we would need the `logging-format-style=old` and 2) we wouldn't be getting the `logging-not-lazy` warnings. Am I right?